components/collector/src/model/entity.py

```diff
@@ -3,14 +3,16 @@
 import re
 from collections.abc import Iterable
 
+from shared.utils.functions import iso_timestamp
+
 QUOTED_SLASH = re.compile("%2f", re.IGNORECASE)
 
 
 class Entity(dict):
     """Class to hold information about an individual measurement entity, e.g. one violation, or one security warning."""
 
     def __init__(self, key: str, **attributes) -> None:
-        kwargs = {"key": self.safe_entity_key(key)}
+        kwargs = {"key": self.safe_entity_key(key), "first_seen": iso_timestamp()}
```

Comment:
I'd expect a `datetime` object, but I guess this is fine since it's never interacted with in the backend (only copied if value is set)?

Reply:
Correct.